requirements.txt

```diff
@@ -5,3 +5,4 @@ whitenoise == 6.0.0
 requests == 2.28.2
 pillow == 9.5.0
 ffmpeg-python == 0.2.0
+eulxml == 1.1.3
```

Comment:
I see this is quite old - June 2016, though they made some changes in Dec 2021 for "Django 3 compatibility".  If it works for us, OK - but I see a lot of forks and old open issues and PRs, so there may be little / no support.

Tests pass, though I get a warning `WARNING: yacc table file version is out of date`.

Reply:
Thanks, i'll update tests and confirm version -- even with the github comments, the MODS schema is ahead of what is currently being used now (mods-3.3 vs mods-3.4), it does offer many convenience wrappers for OAI and MODS, noted. I did not receive the `yacc` warning, i'll try to reproduce after pulling into new test case.

Reply:
For now, using eulxml 1.1.3, but can pull from most[ recent commit ](https://github.com/emory-libraries/eulxml/tree/2e1a) if preferred.
I've set the ply library to a specific version, this should resolve yacc warning.

Reply:
Rebuilding after the `ply` explicit reference, the `yacc` warning is gone.
Let's stick with the official release of `eulxml`; if we run into problems with more extensive use, we can consider pulling from github instead of pypi, or more extreme measures.